hawaii-logging/src/main/java/org/hawaiiframework/logging/config/filter/TransactionTypeFilterConfiguration.java

```diff
@@ -43,20 +47,43 @@ public class TransactionTypeFilterConfiguration {
 
     private static final Logger LOGGER = getLogger(TransactionTypeFilterConfiguration.class);
 
-    @Value("${" + CONFIG_PREFIX + ".order:-700}")
+    @Value("${" + CONFIG_PREFIX + ".order:-1000}")
```

Comment:
why change the default? If required, it has to be changed in the documentation / global configuration as well.